crates/noirc_evaluator/src/ssa/node.rs

```diff
@@ -620,6 +621,39 @@ pub enum BinaryOp {
     Assign,
 }
 
+impl Display for BinaryOp {
```

Comment:
```suggestion
impl std::fmt::Display for BinaryOp {
```

Nitpick but the other implementations of this trait uses the qualified name